lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Release.pm

```diff
@@ -79,7 +79,7 @@ sub serialize
     };
 
     $body{collections} = list_of($entity, $inc, $stash, "collections")
-        if $inc && $inc->collections;
+        if $inc && ($inc->collections || $inc->user_collections);
```

Comment:
Shouldn't need to, since this is just for serializing the data once it's loaded. Controller::WS::2::Release already runs & authenticates for us, then passes things over to here.

The XMLSerializer didn't need any changes because it just checks if any collections were loaded (and they wouldn't have been if those inc params weren't provided): https://github.com/metabrainz/musicbrainz-server/blob/c707e8e264425d2882396cb05ff7a553276ebe6b/lib/MusicBrainz/Server/WebService/XMLSerializer.pm#L444-L444

The JSON behaves differently 'cause it'll output an empty list even if none were found.